### DISCUSSION TITLE
Issue with location of IAEA phsp file output plane in DOSXYZnrc

### DISCUSSION THREAD

**Discussion by bsarey** [nrc-cnrc/EGSnrc#1176] [Questions] [answered]
Dear Community,

I want to use a IAEA format phsp file (source 2) for dose calculation on a ct phantom created with ctcreate. The particles were originally scored at z= 50 cm in the Geant4 framework. With the isocentre being at (0,0,0) I want to irradiate the phantom from the front (see picture below, taken in the XY-plane). The manual states that in the case of IAEA phsp files I have to consider the parameter dsource (distance to isocentre in the gui) as the SAD. I tried different distance values such as 0 cm , +/- 50/60 cm but every time it seemed like I am not hitting the phantom since in dosxyz_show there are no isodose lines visible at all.

Can you please give me some ideas or tricks that I should reconsider?

Thank you

![Bild 25 08 24 um 15 20](https://github.com/user-attachments/assets/636b2a00-9f13-4f85-9621-4aac0df33867)

**Comment by rtownson** [comment]
Consider using beamdp_gui to analyze the phase-space to list the particle properties, this might give you an idea of their directions. Are they headed in the -z direction? It's likely that they're headed in +z, possibly aimed away from your phantom. You'll then have to think about how to rotate the source to account for this properly.

**Reply by bsarey** [reply, marked as answer]
Thanks, I actually have to set the distance to the isocenter as z= 50 cm. Then the particles will be set free exactly at the isocenter. The reason why I couldn't see any deposited dose was that I basically missed the phantom. I didn't consider the angular transformation between the DICOM and DOSXYZnrc coordinate systems.
